packages/actor-init-query/package.json

```diff
@@ -60,9 +60,6 @@
     "readable-stream": "^4.5.2",
     "yargs": "^17.7.2"
   },
-  "optionalDependencies": {
```

Comment:
Why was this removed?

Reply:
I could not get depcheck to stop complaining about a missing `process` dependency when it was located in optional ones, so I moved it to non-optional. :sweat:

Reply:
But it's totally gone now?

Also, something probably changed in depcheck then due to your changes.

Reply:
It is actually still in `dependencies` above, but it was there already previously. This is strange. :thinking: